src/pybind/mgr/cephadm/upgrade.py

```diff
@@ -176,6 +177,26 @@ def _check_target_version(self, version: str) -> Optional[str]:
 
         return None
 
+    def upgrade_ls(self) -> List[str]:
+        reg_name, image = self.mgr.container_image_base.split('/', 1)
+        self.mgr.log.info(f'reg_name {reg_name} image {image}')
+        reg = Registry(reg_name)
+        versions = []
+        for t in reg.get_tags(image):
```

Comment:
I feel like it would be nice to include the registry name in the output of this command. Something like "Ceph versions found on quay.io/ceph/ceph". Especially since we gather and log that info right above this.